staging/src/k8s.io/component-base/logs/json/json.go

```diff
@@ -46,6 +47,48 @@ type zapLogger struct {
 // implement logr.Logger
 var _ logr.Logger = &zapLogger{}
 
+// getCallerFrame gets caller frame. The argument skip is the number of stack
+// frames to ascend, with 0 identifying the caller of getCallerFrame. The
+// boolean ok is false if it was not possible to recover the information.
+//
+// Note: This implementation is similar to runtime.Caller, but it returns the whole frame.
+func getCallerFrame(skip int) (frame runtime.Frame, ok bool) {
+	const skipOffset = 2 // skip getCallerFrame and Callers
+
+	pc := make([]uintptr, 1)
+	numFrames := runtime.Callers(skip+skipOffset, pc)
+	if numFrames < 1 {
+		return
+	}
+
+	frame, _ = runtime.CallersFrames(pc).Next()
+	return frame, frame.PC != 0
+}
+
+// try and set the Caller attribute of the entry,
+// if not possible, return silently.
+func attemptSettingCaller(skip int, entry *zapcore.Entry) {
```

Comment:
I think more standard is `try` prefix.
```suggestion
func trySettingCaller(skip int, entry *zapcore.Entry) {
```